### DISCUSSION TITLE
Meaning of the `n` field in `player_valuations`

### DISCUSSION THREAD

**Discussion by Rithari** [dcaribou/transfermarkt-datasets#281] [Q&A] [answered]
Hello all,

I'm performing some analysis on the dataset for education purposes at my University. I've been banging my head against trying to figure out the meaning behind the column `n` in the `player_valuations` collection.  Could anyone clarify that for me, or, in case I have missed it, is there documentation explaining the dataset in detail somewhere? 

Thank you in advance.

**Comment by dcaribou** [comment, marked as answer]
Hey @Rithari, where are you pulling the data from? I think you might be looking at an outdated version of it.
I sometimes use an auxiliary `n` column to deduplicate records in the data prep stage

https://github.com/dcaribou/transfermarkt-datasets/blob/3fdf249a8af9fa2cb7da3d0038e1cfa4328616b1/dbt/models/base/transfermarkt_api/base_market_value_development.sql#L38

but this has no analytical value, hence ti should be excluded in the `player_valuations` asset.

**Reply by Rithari** [reply]
Hey @dcaribou, thanks for getting back to me so quickly.

The data was provided to me in a university assignment, I suppose it was downloaded at a point in time where that data had that auxiliary column? I unfortunately wouldn't be able to tell you where or how the data was pulled. 

Thanks again for clarifying, I'll be dropping the column in my instance.
